scripts/run.sh

```diff
@@ -39,13 +39,21 @@ then
     LGID="LOCAL_GID=`id -g $USER`"
     [ "$LGID" == "LOCAL_GID=0" ] && LGID="LOCAL_GID=65534"
     mkdir -p $ENV_DIR
+    chmod 0755 $OUTPUT_DIR
```

Comment:
You should not need to enlarge permissions (for security reasons), but rather change by yourself the owner of this directory to the one running your containers.
So this line should not be needed.

Reply:
removed